nemo/collections/asr/parts/manifest.py

```diff
@@ -74,10 +76,33 @@ def item_iter(
 
                 yield item
 
+def __parse_open_stt(line: str, manifest_file: str) -> Dict[str, Any]:
+    manifest_path, _ = split(manifest_file)
+    line_list = line.split(",")
+    assert len(line_list)==3, f"Incorrect fields number: {len(line_list)}"
+    item = dict(
+        offset=None,
+        speaker=None,
+        orig_sr=None,
+    )
+
+    item['audio_file'] = line_list[0]
+    if item['audio_file'][0] != "/":
+        item['audio_file'] = join(manifest_path, item['audio_file'])
+
+    text_file = line_list[1]
+    if text_file[0] != "/":
+        text_file = join(manifest_path, text_file)
+    with open(text_file, 'r') as f:
+            item['text'] = f.read().replace('\n', '')
+
+    item['duration'] = float(line_list[2])
+
+    return item
 
 def __parse_item(line: str, manifest_file: str) -> Dict[str, Any]:
     item = json.loads(line)
-
+    manifest_path, _ = split(manifest_file)
```

Comment:
Could you explain why this is necessary to change ?

Reply:
Together with the line item['audio_file'] = join(manifest_path, item['audio_file']) you can hve inside manifest file relative paths to manifest position

Reply:
Hmm generally we try for absolute paths, but makes sense.